bc_obps/reporting/service/report_non_attributable_service.py

```diff
@@ -0,0 +1,33 @@
+from typing import List
+
+from reporting.models import ReportNonAttributableEmissions, GasType, EmissionCategory
+
+from reporting.schema.report_non_attributable_emissions import ReportNonAttributableIn
+
+
+class ReportNonAttributableService:
+    @classmethod
+    def get_report_non_attributable_by_version_id(cls, report_version_id: int) -> List[ReportNonAttributableEmissions]:
+        return list(ReportNonAttributableEmissions.objects.filter(report_version__id=report_version_id))
```

Comment:
Now that I'm looking at this service, I'm realizing that we're going to need to add `facility_id` to the `report_non_attributable_emissions model` & use that in combination with the `version_id` to fetch these records. Otherwise, when we implement this page for LFOs we won't be able to differentiate which facility these non-attributable emissions belong to with only the `version_id` foreign key